src/nvim/vvars.lua

```diff
@@ -878,6 +878,21 @@ M.vars = {
       <
     ]=],
   },
+  versionlong = {
+    type = 'integer',
+    desc = [=[
+      Like v:version, but also including the patchlevel in the last
+      four digits.  Version 8.1 with patch 123 has value 8010123.
+      This can be used like this: >
+        if v:versionlong >= 8010123
+      <
+      However, if there are gaps in the list of patches included
```

Comment:
It seems that to match upstream docs, this needs to be changed to this (there is a Tab before the `if`, not two spaces, and there are two Tabs before the "However"):
```suggestion
      	if v:versionlong >= 8010123
      <		However, if there are gaps in the list of patches included
```
as `render_vvar_doc()` uses a line literally if it starts with `>`:
```lua
      if l == '<' or l:match('^<%s') then
        write(l)
      else
        write('\t\t' .. l:gsub('\\<', '<'))
      end
```

Reply:
I grouped the output of `vim-patch.sh -p` of (NA) patches in 1 commit and inserted a note between `Co-author` and patch description.

> The Co-authored-by: line needs to be at the bottom of the commit message, otherwise GitHub won't recognize it.

If you mean to also group these all at the bottom, then sure I can rebase again to update them all.

Reply:
Actually, I think the `Co-authored-by:` lines of N/A patches should be removed, and the `Co-authored-by:` line of the non-N/A patch should be moved to the bottom.

Reply:
This is not a big deal. Let's just leave as-is.